### DISCUSSION TITLE
Waitlist for Attendees

### DISCUSSION THREAD

**Discussion by way0utwest** [dataplat/DataSaturdays#169] [Ideas]
With the new COVID world, and potential limitations on attendance when things open up, I would think a way for attendees to be added to a waitlist, by time of registration. I would include the ability for someone to cancel a registration and auto move of people from the waitlist to the attendee list automatically.

Organizers ought to be able to bulk move xx people as well.

**Comment by sqlzelda** [comment]
Often times, sponsors did not register for events. This would usually cause them to be wait listed. I'm not sure we want to continue with that model, but it may be something to think about as we design out the attendee functionality and/or count.
